bigquery_etl/cli/glean_usage.py

```diff
@@ -4,7 +4,7 @@
 from pathlib import Path
 import click
 
-from ..cli.utils import (
+from ..cli.common import (
```

Comment:
why is this being renamed?

Reply:
probably not necessary. I'll undo the rename.